objmodel/chapter.md

```diff
@@ -232,15 +232,14 @@ TYPE.cls = TYPE
 OBJECT.cls = TYPE
 ````
 
-To define new metaclasses it is enough to subclass ``TYPE``. However, in the
-rest of this chapter we won't do that, and simply always use ``TYPE`` as the
+To define new metaclasses, it is enough to subclass ``TYPE``. However, in the
+rest of this chapter we won't do that; we'll simply always use ``TYPE`` as the
 metaclass of every class.
 
-XXX diagram: figures/inheritance.svg
+![](figures/inheritance.png)
```

Comment:
Yes, the current conversion somehow lost the arrowheads on the lines, so we must make better ones later.